Diagnostics/Get-NetView.PS1

```diff
@@ -936,11 +936,11 @@ function ChelsioDetail {
         $out = (Join-Path -Path $dirBus -ChildPath $file)
         $OutFlash = (Join-Path -Path $dirBus -ChildPath $flashFile)
         [String []] $cmds = "cxgbtool.exe $ifNameVbd hardware sgedbg",
-                            "cxgbtool.exe $ifNameVbd hardware flash $OutFlash"
+                            "cxgbtool.exe $ifNameVbd hardware flash ""$OutFlash"""
         foreach ($cmd in $cmds) {
             ExecCommand -Command $cmd -Output $out
         }
-    }
+    } # -not $Global:ChelsioRunOnce
```

Comment:
This comment should be on 942,; the closing IF() basic block, rather than line 943 which is the function ending line.  As is, the comment will be misleading.